xml/defaultConfig.xml

```diff
@@ -112,4 +112,23 @@
         <!-- <treeTransportMission></treeTransportMission> -->
         <!-- <destructibleRockMission></destructibleRockMission> -->
     </customRewards>
+
+    <customMaxType>
+        <!-- <baleMission></baleMission> -->
+        <!-- <baleWrapMission></baleWrapMission> -->
+        <!-- <plowMission></plowMission> -->
+        <!-- <cultivateMission></cultivateMission>  -->
+        <!-- <sowMission></sowMission> -->
+        <harvestMission>12</harvestMission>
```

Comment:
The values in the default config will get applied to every player who installs the mod - I'd prefer slightly more sane values here for the defaults... I'm fine with adding an override for harvest missions to something like 8-10, but not the default overrides to "lower" the default values for `deadwoodMission`, `treeTransportMissions`, or `destructibleRockMission` as many players love those type of missions. 

If you're in the code, we'd also need documentation added to the README.md file regarding this new configuration setting.